src/core/manager.c

```diff
@@ -3482,9 +3482,9 @@ static void manager_send_ready(Manager *m) {
                 /* Skip the notification if nothing changed. */
                 return;
 
-        r = sd_notifyf(false,
-                       "%sSTATUS=Ready.",
-                       m->ready_sent ? "READY=1\n" : "");
+        r = sd_notify(false,
+                      "READY=1\n"
+                      "STATUS=Ready.");
```

Comment:
nit: should fit into a single line?

Reply:
It's multi-line on purpose: to make it clear that there are two separate lines being sent.

Reply:
ok, makes sense